core/integration/client_tests.go

```diff
@@ -492,3 +501,172 @@ func (m uint64Slice) Less(i, j int) bool { return m[i] < m[j] }
 func cp(tag int) []byte {
 	return []byte(fmt.Sprintf("bar%v", tag))
 }
+
+// GenerateTestVectors verifies set/get semantics.
+// TODO(gbelvin): Migrate into core/integration/client_tests to avoid code rot.
+func GenerateTestVectors(ctx context.Context, env *Env) error {
```

Comment:
This function is largely a duplicate of `TestEmptyGetAndUpdate`. If we could merge the two that would be ideal - or we can save that for a follow on PR.